backend/src/main/java/com/daedan/festabook/timetag/infrastructure/TimeTagJpaRepository.java

```diff
@@ -1,10 +1,13 @@
 package com.daedan.festabook.timetag.infrastructure;
 
 import com.daedan.festabook.timetag.domain.TimeTag;
+import java.util.Collection;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface TimeTagJpaRepository extends JpaRepository<TimeTag, Long> {
 
     List<TimeTag> findAllByFestivalId(Long festivalId);
+
+    List<TimeTag> findAllByIdIn(Collection<Long> ids);
```

Comment:
```suggestion

    List<TimeTag> findAllById(Collection<Long> ids);
```

in 빼고 작성해도 될 것 같은데 어때요?